src/core/localization/linksNoTranslation.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, expect, it, test } from 'vitest';
+import gettextParser from 'gettext-parser';
+import fs from 'fs';
+
+// Here the input data for the test is specified
+const languageCodes = ['ar', 'de', 'es', 'id', 'ko', 'uk'];
+
+const processLinkAndText = (linkAndText) => {
```

Comment:
Better name would be something like "parseLinkAndText" or "extractLinkAndText". The word "process" isn't really describing what the function is doing.

But also - you probably shouldn't be doing the parsing by hand, there're libraries for markdown parsing, for example:
[mdast-util-from-markdown](https://www.npmjs.com/package/mdast-util-from-markdown)

Reply:
imported and used library in a separate function